ocs_ci/utility/localstorage.py

```diff
@@ -32,3 +39,114 @@ def fetch_all_device_paths():
     os.chdir(constants.TOP_DIR)
     shutil.rmtree(path)
     return dev_paths
+
+
+def get_new_device_paths(device_sets_required, osd_size_capacity_requested):
+    """
+    Get new device paths to add capacity over Baremetal cluster
+
+    Args:
+        device_sets_required (int) : Count of device sets to be added
+        osd_size_capacity_requested (int) : Requested OSD size capacity
+
+    Returns:
+        list : List containing added device paths
+
+    """
+    ocp_obj = OCP()
+    workers = get_typed_nodes(node_type="worker")
```

Comment:
general note - please add more comments in the code. It is hard to follow everything that is going on

Reply:
Added!

Reply:
thanks!